### ISSUE TITLE
Convergence failed for the WideResNet model

### ISSUE THREAD

**Issue by baochunli** [TL-System/plato#1]
With the current configuration for the WideResNet model, training does not converge. Things that need to be added or fixed:

- Add a learning rate schedule in addition to the optimizer;
- Review (and compare) the training procedure in https://github.com/xternalz/WideResNet-pytorch/.
